src/witan/models/dem/ccm/fert/fertility.clj

```diff
@@ -138,6 +183,26 @@
                           :fert-rate :births)]
     {:births-by-age-sex-mother projected-births}))
 
+(defworkflowfn project-births-1-1-0
```

Comment:
@sunnytownsend I can't remember my own advice now. I think changing schema across versions is discouraged. This has different input schema.